test/integration_test.py

```diff
@@ -19,6 +19,8 @@
 from hydra import compose, initialize
 from omegaconf import DictConfig, OmegaConf
 
+from test.conftest import find_replace
```

Comment:
I'm not sure if this is good practice to import from `conftest` can't we just move this function into the `test/integration_test.py` or create a file for helper fcts?

Reply:
Ye agreed, it felt a bit weird to me, I'll move it :smile: